charts/rsyncd/tests/custom_values_test.yaml

```diff
@@ -16,21 +44,190 @@ tests:
+          value: rsyncd-conf
+      - equal:
+          path: spec.template.spec.volumes[0].configMap.name
+          value: RELEASE-NAME-rsyncd-conf
+      - equal:
+          path: spec.template.spec.containers[0].volumeMounts[0].name
+          value: rsyncd-conf
+      - equal:
+          path: spec.template.spec.containers[0].volumeMounts[0].mountPath
+          value: /etc/rsyncd.d/rsyncd.inc
+      # Custom motd mounted
+      - equal:
+          path: spec.template.spec.volumes[1].name
+          value: jenkins-motd
+      - equal:
+          path: spec.template.spec.volumes[1].configMap.name
+          value: RELEASE-NAME-rsyncd-jenkins-motd
+      - equal:
+          path: spec.template.spec.containers[0].volumeMounts[1].name
+          value: jenkins-motd
+      - equal:
+          path: spec.template.spec.containers[0].volumeMounts[1].mountPath
+          value: /tmp/motd
+      # Custom Rsyncd components
+      - equal:
+          path: spec.template.spec.volumes[2].name
+          value: datadir-jenkins
+      - equal:
+          path: spec.template.spec.volumes[2].persistentVolumeClaim.claimName
+          value: super-pvc
+      - equal:
+          path: spec.template.spec.containers[0].volumeMounts[2].name
+          value: rsyncd-conf
+      - equal:
+          path: spec.template.spec.containers[0].volumeMounts[2].mountPath
+          value: /etc/rsyncd.d/jenkins.conf
+      - equal:
+          path: spec.template.spec.containers[0].volumeMounts[2].subPath
+          value: jenkins.conf
+      - equal:
+          path: spec.template.spec.containers[0].volumeMounts[2].readOnly
+          value: true
+      - equal:
+          path: spec.template.spec.containers[0].volumeMounts[3].name
+          value: datadir-jenkins
+      - equal:
+          path: spec.template.spec.containers[0].volumeMounts[3].mountPath
+          value: /rsyncd/data/jenkins
+      - notExists:
+          path: spec.template.spec.containers[0].volumeMounts[3].subPath
+      - equal:
+          path: spec.template.spec.containers[0].volumeMounts[3].readOnly
+          value: true
+  - it: should create a custom motd config map
+    template: configmap.jenkins-motd.yaml
+    set:
+      configuration:
+        motd:
+          content: |-
+            ==========
+            FOO BAR
+            SUPER MOTD
+            ==========
+    asserts:
+      - hasDocuments:
+          count: 1
+      - isKind:
+          of: ConfigMap
+      - equal:
+          path: metadata.name
+          value: RELEASE-NAME-rsyncd-jenkins-motd
+      - notMatchRegex:
+          path: data["jenkins.motd"]
+          pattern: JENKINS MIRROR
+      - matchRegex:
+          path: data["jenkins.motd"]
+          pattern: SUPER MOTD
+  - it: should create a custom rsyncd-conf config map
+    template: configmap.rsyncd-conf.yaml
+    set:
+      configuration:
+        motd:
+          path: /tmp/motd
+        components:
+          - name: jenkins
```

Comment:
I would add a test with 2 components.

Reply:
Ref: https://github.com/jenkins-infra/helm-charts/pull/665/files#r1317715286

Reply:
> Ref: https://github.com/jenkins-infra/helm-charts/pull/665/files#r1317715286

Unrelated (see my comment) but worth testing with 2 components yes

Reply:
Implemented in https://github.com/jenkins-infra/helm-charts/pull/665/commits/8ca5d5bad34d45b8793e2f67358567c01cfcbe0d